fastlane/spec/actions_specs/modify_services_spec.rb

```diff
@@ -24,7 +24,7 @@
                 associated_domains: 'off',
                 wallet: :on,
                 apple_pay: :off,
-                data_protection: true,
```

Comment:
This was the test if `true` is changed to `on` I guess. You should add another service to the list that is set to `true` so this behavior will still be tested after your changes.